lib/rubocop/cop/style/ip_addresses.rb

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+require 'resolv'
+
+module RuboCop
+  module Cop
+    module Style
+      # Checks for IP addresses in strings
```

Comment:
Could you please justify this cop here for the manual? Your issue said:

> IP addresses being fixed in your code is generally a quick way to make something that's non-reusable, brittle, requires a new commit/deploy when the IP changes instead of ENV configuration/relaunch, and to have a failed deploy because something changed.

Reply:
It's not actually my issue I just thought I could tackle it. Should I just put that into the comments?

Reply:
Oops. I didn't compare usernames. Sorry about that.

I think it could be rephrased for simplicity. @tibbon, would you like to help rephrase this?

Reply:
I edited it down and amended the comment.